node/async-api-converter/.editorconfig

```diff
@@ -0,0 +1,4 @@
+# 4 space indentation
+[*.{js} ]
+indent_style = space
+indent_size = 4
```

Comment:
Do we still need this kind of file since we have it on the root level also?

Reply:
removed :)